cfpb/consumerfinance.gov
# Text introduction and related content: Add more missing aria_label to templates

https://github.com/cfpb/consumerfinance.gov/pull/6497 added an aria-label to links edited in Wagtail, but neglected to add the aria-label to the text introduction and related content templates, so the entered aria-labels don't show up in those cases.

## Changes

- Add aria-label to the text introduction template.
- Add aria-label to the related content template.
- Add the aria_label to the code comments, and format comments to align new values width.



## How to test this PR

1. Edit http://localhost:8000/consumer-tools/mortgages/ and add a test link and aria-label to the text-introduction and to the related content links, preview the page and inspect the links to see that they have aria-attributes.
